docs/recipes/ConfiguringYourStore.md

```diff
@@ -0,0 +1,311 @@
+
+```js
+import { applyMiddleware, createStore } from 'redux'
+import thunkMiddleware from 'redux-thunk'
+import { composeWithDevtools } from 'redux-devtools-extension'
+
+import monitorReducersEnhancer from './enhancers/monitorReducers'
+import loggerMiddleware from './middleware/logger'
+import rootReducer from './reducers'
+
+export default function configureStore(preloadedState) {
+  const middlewares = [loggerMiddleware, thunkMiddleware]
+  const middlewareEnhancer = applyMiddleware(...middlewares)
+
+  const enhancers = [middlewareEnhancer, monitorReducersEnhancer]
+  const composedEnhancers = composeWithDevtools(...enhancers)
+
+  const store = createStore(rootReducer, preloadedState, composedEnhancers)
+
+  return store
+}
+```
+
+And that's it!
+
+If we now visit our app via a browser with the devtools extension installed, we can explore and debug using a powerful new tool.
+
+## Hot reloading
+
+Another powerful tool which can make the development process a lot more intuitive is hot reloading, which means replacing pieces of code without restarting your whole app.
+
+For example, consider what happens when you run your app, interact with it for a while, and then decide to make changes to one of your reducers. Normally, when you make those changes your app will restart, reverting your Redux state to its initial value.
+
+With hot module reloading enabled, only the reducer you changed would be reloaded, allowing you to change your code _without_ resetting the state every time. This makes for a much faster development process.
+
+We'll add hot reloading both to our Redux reducers and to our React components.
+
+First, let's add it to our `configureStore` function:
+
+```js
+import { applyMiddleware, compose, createStore } from 'redux'
+import thunkMiddleware from 'redux-thunk'
+
+import monitorReducersEnhancer from './enhancers/monitorReducers'
+import loggerMiddleware from './middleware/logger'
+import rootReducer from './reducers'
+
+export default function configureStore(preloadedState) {
+  const middlewares = [loggerMiddleware, thunkMiddleware]
+  const middlewareEnhancer = applyMiddleware(...middlewares)
+
+  const enhancers = [middlewareEnhancer, monitorReducersEnhancer]
+  const composedEnhancers = compose(...enhancers)
+
+  const store = createStore(rootReducer, preloadedState, composedEnhancers)
+
+  if (process.env.NODE_ENV !== 'production' && module.hot) {
+    module.hot.accept('./reducers', () =>
+      store.replaceReducer(rootReducer)
+    )
+  }
+
+  return store
+}
+```
+
+The new code is wrapped in an `if` statement, so it only runs when our app is not in production mode, and only if the `module.hot` feature is available.
+
+We use Webpack's `module.hot.accept` method to specify which module should be hot reloaded, and what should happen when the module changes. In this case, we're watching the `./reducers` module, and passing the updated `rootReducer` to the `store.replaceReducer` method when it changes.
```

Comment:
Since parcel is becoming more popular and also [supports the same API](https://parceljs.org/hmr.html) this should not only mention webpack.

Reply:
Sure, I'll go ahead and edit that real fast.

Reply:
Done.  Reworded to: